dvc/exceptions.py

```diff
@@ -178,6 +178,13 @@ def __init__(self):
         )
 
 
+class NoMetricsParsedError(DvcException):
+    def __init__(self,):
+        super().__init__(
+            "Could not parse any metric. Use `-v` to investigate."
```

Comment:
`Could not parse metrics files. Use `-v` option to see more details.`

cc @jorgeorpinel - let's get this message right

Reply:
Fixed the message

Reply:
Hey sorry guys just noticed this mention. I did suggest "The metrics/plots file could not be parsed." before in https://github.com/iterative/dvc/pull/4370#discussion_r468221756 though. I don't think we need to mention -v, that applies to every error message.